### PR TITLE
Polish the readme doc

### PR DESCRIPTION
**Pull request by leVirve**
Here are some changes and new title doc for readme, would you mind if I changed it or needs other modifications? thanks 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,55 +55,70 @@ sample_rate = 44100
...
+<img src="screenshots/Demo.gif">
 
 ## Tweaks
-To show more images in tensorboard's image tab, just
-modify the hardcoded `event_accumulator` in
-`~/anaconda3/lib/python3.6/site-packages/tensorflow/tensorboard/backend/application.py`
-as you wish.
 
-For tensorflow-tensorboard>0.17 see https://github.com/lanpa/tensorboard-pytorch/issues/44
+*built-in feature in `tensorflow-tensorboard 0.4rc`.*
```
Comment (lanpa) [inline README.md]: which feature?

File: README.md [hunk 2]
```diff
@@ -40,55 +55,70 @@ sample_rate = 44100
...
-    writer.add_scalar('data/scalar1', s1[0], n_iter) #data grouping by `slash`
-    writer.add_scalar('data/scalar2', s2[0], n_iter)
-    writer.add_scalars('data/scalar_group', {"xsinx":n_iter*np.sin(n_iter),
-                                             "xcosx":n_iter*np.cos(n_iter),
-                                             "arctanx": np.arctan(n_iter)}, n_iter)
-    x = torch.rand(32, 3, 64, 64) # output from network
-    if n_iter%10==0:
-        x = vutils.make_grid(x, normalize=True, scale_each=True)
+
+    ''' scalar '''
```
Comment (lanpa) [inline README.md]: I think the function name is self-explained

File: README.md [hunk 3]
```diff
@@ -1,32 +1,47 @@
+# tensorboard X
```
Comment (lanpa) [inline README.md]: no space

File: README.md [hunk 4]
```diff
@@ -1,32 +1,47 @@
...
+## Demo
 
-If the demo code `onnx_graph.py` is not working, you have to build pytorch and onnx from source.
+[demo site](http://35.197.26.245:6006) (TensorBoard for `demo.py` and some images generated by BEGAN)
 
-## Install
+*If the demo code `onnx_graph.py` is not working, you have to build pytorch and onnx from source.*
 
-`#tested on anaconda2/anaconda3, pytorch 0.3, torchvision 0.2`
+## Requirement
```
Comment (lanpa) [inline README.md]: not strict requirement (you can use without anaconda, without pytorch, etc)

File: README.md [hunk 5]
```diff
@@ -1,32 +1,47 @@
...
+## Install
+
+`pip install tensorboardX`
 
 or build from source:
+
 `pip install git+https://github.com/lanpa/tensorboard-pytorch`
 
 ## API
-http://tensorboard-pytorch.readthedocs.io/en/latest/tensorboard.html
```
Comment (lanpa) [inline README.md]: I moved it to the top, treat it as project website

File: README.md (hunk 1 above)
Comment (leVirve) [reply README.md]: To control show how many images in Tensorboard's image tab. Does this description state this feature? Or I misunderstand :joy:

image

Comment (lanpa) [reply README.md]: That should be about changing the number of visualizations per page.
As for 'show more images', I mean add more ticks for the slider. XD

File: README.md (hunk 5 above)
Comment (lanpa) [reply README.md]: I did it~

File: README.md (hunk 1 above)
Comment (leVirve) [reply README.md]: Okay, I see XD

**Pull request merged**